App/Utilities/Config/RealmConfig.swift

```diff
@@ -11,6 +11,13 @@ import RealmSwift
 var realmConfig: Realm.Configuration {
     let container = FileManager.default.containerURL(forSecurityApplicationGroupIdentifier: "group.com.example.tumble")
     let realmURL = container?.appendingPathComponent("realm.db")
-    let config = Realm.Configuration(fileURL: realmURL, schemaVersion: 1)
+    let config = Realm.Configuration(fileURL: realmURL, schemaVersion: 2, migrationBlock: { migration, oldSchemaVersion in
```

Comment:
Have you verified that the widget still works properly?

Reply:
I have updated the app in sumulator and added the widget to homescreen. No errors occured